### PR TITLE
Update OAuth2.0 example. OKTA-339587

### PR DESCRIPTION
**Pull request by serhiibuniak-okta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,20 +99,79 @@ okta_client = OktaClient()
...
-This SDK supports this feature (OAuth 2.0) only for service-to-service applications. Check out [our guides][oauth-guides] to learn more about how to register a new service application using a private and public key pair.
+This SDK supports this feature (OAuth 2.0) only for service-to-service applications. Check out [our guides](https://developer.okta.com/docs/guides/implement-oauth-for-okta-serviceapp/overview/) to learn more about how to register a new service application using a private and public key pair.
 
 When using this approach you won't need an API Token because the SDK will request an access token for you. In order to use OAuth 2.0, construct a client instance by passing the following parameters:
 
 ```py
 from okta.client import Client as OktaClient
 config = {
-    'orgUrl': 'https://test.okta.com',
+    'orgUrl': 'https://YOUR_DOMAIN',
```
Comment (bretterer) [inline README.md:109]: change to `https://{yourOktaDomain}` to match the standard

File: README.md [hunk 2]
```diff
@@ -99,20 +99,79 @@ okta_client = OktaClient()
...
     'authorizationMode': 'PrivateKey',
     'clientId': 'YOUR_CLIENT_ID',
```
Comment (bretterer) [inline README.md:111]: Update to `{yourClientId}`

**Review by bretterer — COMMENTED** [review]
Couple changes to make. after that LGTM

**Review by bretterer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (serhiibuniak-okta) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (serhiibuniak-okta) [reply README.md]: done

**Pull request merged**